packages/compartment-mapper/test/bundle.test.js

```diff
@@ -69,6 +69,54 @@ test('bundles work', async t => {
   t.deepEqual(log, expectedLog);
 });
 
+test('using named evaluate bundles work', async t => {
```

Comment:
These tests are very helpful in understanding the new functionality, but documentation would also be appreciated.